src/modules/commander/Commander.cpp

```diff
@@ -574,7 +574,9 @@ transition_result_t Commander::arm(arm_disarm_reason_t calling_reason, bool run_
 
 			if (!_vehicle_control_mode.flag_control_climb_rate_enabled &&
 			    !_failsafe_flags.manual_control_signal_lost && !_is_throttle_low
-			    && !is_ground_vehicle(_vehicle_status)) {
+			    && ((_vehicle_status.vehicle_type == vehicle_status_s::VEHICLE_TYPE_FIXED_WING)
```

Comment:
Please don't use the _vehicle_status.vehicle_type in Commander - Commander has a notion of `system_type` (which checks the MAV_TYPE), use that to infer controller logic.

Reply:
@sfuhrer @dagar what is the consensus on this?

Reply:
@sfuhrer if there is no further objections, can we proceed with the merge? In case something is raised against it, I am happy to fix it.